tests/compute/test_heterograph.py

```diff
@@ -365,6 +365,40 @@ def _test(g):
     # test repr
     print(g)
 
+@parametrize_dtype
+def test_empty_query(idtype):
+    g = dgl.graph(([1, 2, 3], [0, 4, 5]), idtype=idtype, device=F.ctx())
+    g.add_nodes(0)
+    g.add_edges([], [])
+    g.remove_edges([])
+    g.remove_nodes([])
+    assert F.shape(g.has_nodes([])) == (0,)
+    assert F.shape(g.has_edges_between([], [])) == (0,)
```

Comment:
MXNet do not support summation on a empty array: https://github.com/dmlc/dgl/blob/master/python/dgl/heterograph.py#L2568, resulting in this error.